src/routes/middleware/acl.ts

```diff
@@ -0,0 +1,93 @@
+import { exec, ExecException } from 'child_process';
+import { ACL_FILES_PATH, ACL_TOOL_PATH } from '../../config';
+import { promisify } from 'util';
+import e, { NextFunction, Request, Response, Router } from 'express';
+import {
+  fromString as fromHexString,
+} from 'hex-array';
+const execPromise = promisify(exec);
+
+export async function ensureAclPassed(req: Request, res: Response, next: NextFunction) {
+  console.log("[ACL] Control reached middleware ensureAclPassed req.headers.cookie value", req.headers.cookie);
```

Comment:
same here, thank you!

Reply:
But maybe... Let's keep the logs while we are testing it.
Just let's make sure it will not be too massive: having 6-7 lines of logs every 5 seconds will blow up the log file